README.md

```diff
@@ -268,10 +269,12 @@ POLICY_DOC='{
 }'
 aws --profile="$AWS_PROFILE" iam create-policy --policy-name "$LOKI_POLICY" --policy-document "$POLICY_DOC"
 ```
-* create a new IAM Role that allows the necessary instances (k8s masters in the
-  case of using `kiam`) to access resources from the policy. Set trust to allow
-  the Role used by `kiam` to claim the S3 access role.
-  If you want to use AWS WebUI, copy/paste the contents of `POLICY_DOC` variable.
+
+#### Prepare AWS IAM role
+
+**Up to giantswarm v18**
```

Comment:
Subsection title (`####`) and bold text (`**`) look quite the same.
![image](https://github.com/giantswarm/loki-app/assets/12008875/d585e7d1-5126-4b4f-9cef-af90e5d17ea6)

Maybe it would be better as a sub-subsection (`#####`)?

Reply:
Strange, on my local preview they don't look the same : 
![image](https://github.com/giantswarm/loki-app/assets/64951262/8ef25424-6da9-426f-b401-36e5d0a2489f)